wfe-core/src/main/java/ru/runa/wfe/chat/socket/ChatSocketMessageHandler.java

```diff
@@ -2,11 +2,12 @@
 
 import java.io.IOException;
 import javax.websocket.Session;
+import ru.runa.wfe.chat.dto.ChatDto;
 import ru.runa.wfe.user.User;
 
-public interface ChatSocketMessageHandler {
+public interface ChatSocketMessageHandler<T extends ChatDto> {
 
-    void handleMessage(Session session, String objectMessage, User user) throws IOException;
+    void handleMessage(Session session, T dto, User user) throws IOException;
 
-    boolean checkType(String messageType);
+    boolean isSupports(Class<? extends ChatDto> messageType);
```

Comment:
```suggestion
    boolean isSupports(Class<T> messageType);
```
По идее так даже правильнее